src/libsync/propagatedownload.cpp

```diff
@@ -850,6 +874,18 @@ void PropagateDownloadFile::deleteExistingFolder()
     }
 }
 
+void PropagateDownloadFile::startContentChecksumCompute(const QByteArray &checksumType, const QString &path)
+{
+    qCInfo(lcPropagateDownload) << "Start checksum compute with checksumType:" << checksumType << " for path:" << path;
+    // Compute the content checksum.
+    auto computeChecksum = new ComputeChecksum(this);
```

Comment:
Could be const I think?

Reply:
@FlexW Nah. Later I call non-const methods. This won't work. Unless you want a **const pointer** to a **non-const** object.

Reply:
yes, just thought to make the pointer const. But okay, I will no fight for it.